src/theme.ts

```diff
@@ -141,14 +154,68 @@ const defaultTheme: ThemeInterface = {
         ? theme.sidebar.textColor
         : theme.colors.primary.main,
     groupItems: {
+      backgroundColor: 'transparent',
+      color: ({ sidebar }) => sidebar.textColor,
+      fontSize: '0.8em',
+      fontWeight: 'inherit',
+      opacity: '0.7',
+      active: {
+        color: ({ sidebar }) => sidebar.groupItems.color,
+        backgroundColor: ({ sidebar }) => sidebar.backgroundColor
+      },
+      hover: {
+        color: ({ sidebar }) => sidebar.groupItems.color,
+        backgroundColor: ({ sidebar }) => sidebar.groupItems.backgroundColor
+      },
       textTransform: 'uppercase',
+      arrow: {
+        size: ({ sidebar }) => sidebar.arrow.size,
+        color: ({ sidebar }) => sidebar.arrow.color
+      },
     },
     level1Items: {
+      backgroundColor: ({ sidebar }) => sidebar.groupItems.backgroundColor,
+      color: ({ sidebar }) => sidebar.groupItems.color,
```

Comment:
textColor instead of color to follow the same naming convention in the previous version?